auth-microservices/controller/user.controller.js

```diff
@@ -53,6 +53,36 @@ class UserController {
         .json({ message: "An error occurred while fetching users" });
     }
   }
+  static async changeUserDetail(req, res) {
```

Comment:
Only push the changes related to comment microservice. Delete all the other commits.